app/models/bulkrax/csv_entry.rb

```diff
@@ -124,29 +147,27 @@ def prepare_export_data(datum)
     end
 
     def object_metadata(data, object_key)
-      data = convert_to_hash(data)
-
+      data = data.map { |d| eval(d) }.flatten # rubocop:disable Security/Eval
```

Comment:
I would love a better solution here. I think that converting to json and back is just as dangerous but also more complex.

Reply:
why is it just as dangerous? I'm not familiar with `eval` outside of when @sephirothkod and I were pairing on the code in `convert_to_hash` and he mentioned that `eval` is unsafe.

Reply:
I would also like an answer here, I have always been told that `eval` exists to never be used. ie. it is such a security risk that you should never use it other than for debugging.